pkg/endpoint/endpoint.go

```diff
@@ -128,6 +128,10 @@ type Endpoint struct {
 	// recalculated on endpoint restore.
 	createdAt time.Time
 
+	// lastUpdatedTime store the time the endpoint security identity was last changed.
```

Comment:
This comment seems to need an update as well :)

Reply:
Can you also include what the key and value of the map represent, plus maybe a description of why it's useful to have a map vs. one value? This will help with answering the other questions Weilong posed below.